Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabInstanceUtils.h

```diff
@@ -10,39 +10,61 @@
 
 #include <AzCore/std/string/string.h>
 #include <AzCore/std/utils.h>
-#include <AzToolsFramework/Prefab/Instance/Instance.h>
 
 namespace AzToolsFramework
 {
     namespace Prefab
     {
+        class Instance;
+
+        //! Defines climb-up result returned by climb-up operation.
+        struct InstanceClimbUpResult
+        {
+            //! Instance reference to the destination.
+            InstanceOptionalConstReference reachedInstance = AZStd::nullopt;
```

Comment:
standards: member variables for both classes and structs should start with m_ prefix